pontoon/base/models/user.py

```diff
@@ -445,6 +492,9 @@ def latest_action(self):
 User.add_to_class("locale_role", user_locale_role)
 User.add_to_class("status", user_status)
 User.add_to_class("contributed_translations", contributed_translations)
+User.add_to_class("badges_translations_count", badges_translations_count)
+User.add_to_class("badges_reviewed_translations", badges_reviewed_translations)
+User.add_to_class("badges_promoted_users", badges_promoted_users)
```

Comment:
Nit: For consistency, I'd use these names:

```suggestion
User.add_to_class("badges_translation_count", badges_translation_count)
User.add_to_class("badges_review_count", badges_review_count)
User.add_to_class("badges_promotion_count", badges_promotion_count)
```